packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -6,254 +6,214 @@ const { argTypes, parameters } = storybookDefaults('sd-teaser-media');
 const { overrideArgs } = storybookHelpers('sd-teaser-media');
 const { generateTemplate } = storybookTemplate('sd-teaser-media');
 
+/**
+ * Used as a flexible container that groups related items and links them to further content.
+ *
+ * **Related components**:
+ * - [sd-teaser](?path=/docs/components-teaser--docs)
+ *
+ * **Related templates**:
+ * - [Teaser Media](?path=/docs/templates-teaser-media--docs)
+ * - [Teaser](?path=/docs/templates-teaser--docs)
+ */
 export default {
+  tags: ['!dev'],
   title: 'Components/sd-teaser-media',
   component: 'sd-teaser-media',
   args: overrideArgs([
     {
       type: 'slot',
       name: 'default',
-      value: `<div class="slot slot--border slot--text h-12">Main slot</div>`
+      value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
     },
     {
       type: 'slot',
       name: 'media',
-      value: `<div slot="media" class="slot slot--border slot--text h-[384px]">Media slot</div>`
+      value: `<img slot="media" src="./placeholders/images/generic.jpg" class="aspect-video object-cover" alt="Test"/>`
     },
     {
       type: 'slot',
       name: 'meta',
       value: `<div slot="meta" class="slot slot--border slot--text h-12">Meta slot</div>`
+    },
+    {
+      type: 'slot',
+      name: 'headline',
+      value: `<div slot="headline" class="h-12">Headline Media Teaser</div>`
     }
   ]),
   argTypes,
   parameters
 };
 
-/**
- * This shows sd-teaser-media in its default state.
- */
-
 export const Default = {
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Teaser-Media in all possible `variant`.
+ * Use the `variant` attribute to set the color variant:
+ * - `white (default)`
+ * - `primary`
+ * - `primary-100`
+ * - `neutral-100`
+ * - `gradient-dark`
+ * - `gradient-white`
  */
 
 export const Variant = {
   name: 'Variant',
-  parameters: { controls: { exclude: ['variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'variant' }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px; width: 600px">%TEMPLATE%</div>'
-        },
-        {
-          type: 'slot',
-          name: 'media',
-          value:
-            '<img slot="media" class="aspect-video object-cover" src="./placeholders/images/generic.jpg" alt="A skyline of a city by night"/>'
-        }
-      ]
-    });
-  }
+  render: () => html`
+    <div style="margin-bottom: 40px; width: 600px">
```

Comment:
You can use classes instead of inlining the style. Place all sd-teaser-media components inside a single div wrapper and you can then apply `flex flex-col space-y-* w-[*]`. If it's not clear, feel free to call me.

Reply:
Updated